api/src/main/java/org/eclipse/microprofile/faulttolerance/Asynchronous.java

```diff
@@ -28,7 +28,9 @@
 import javax.interceptor.InterceptorBinding;
 
 /**
- * Wrap the execution and invoke it asynchronously.
+ * Wrap the execution and invoke it asynchronously. Despite of the asychrounous execution, the context for 
+ * {@link javax.enterprise.context.RequestScoped} must be active during the method invocation, 
+ * which means the method with {@code @Asynchronous} annotation is allowed to use the beans with {@link javax.enterprise.context.RequestScoped}.
```

Comment:
"Despite of the asynchronous execution" should be "Despite the asynchronous execution".

However, I think it would be better to take out that part completely and just say `the context for {@link javax.enterprise.context.RequestScoped} must be active during the method invocation`.

`the method with {@code @Asynchronous} annotation` should either be `the method with the {@code @Asynchronous} annotation` or `the method annotated with {@code @Asynchronous}`

Reply:
fixed